man/systemd.network.xml

```diff
@@ -2172,9 +2172,9 @@ Table=1234</programlisting></para>
       <varlistentry>
         <term><varname>WithoutRA=</varname></term>
         <listitem>
-          <para>Allows DHCPv6 client to start without router advertisements's managed or other
-          address configuration flag. Takes one of <literal>no</literal>, <literal>solicit</literal>
-          or <literal>information-request</literal>. If this is not specified,
+          <para>Allows DHCPv6 client to start without router advertisements's "managed" or "other
+          configuration" flag. Takes one of <literal>no</literal>, <literal>solicit</literal> or
```

Comment:
This is not your fault, but could you add a comma after `<literal>solicit</literal>`?

Reply:
Done.